tests/providers/google/cloud/transfers/test_sql_to_gcs.py

```diff
@@ -557,7 +557,7 @@ def test__write_local_data_files_csv_does_not_write_on_empty_rows(self):
         with pytest.raises(StopIteration):
             next(files)["file_handle"]
 
-        assert len([f for f in files]) == 0
+        assert not list(files)
```

Comment:
```suggestion
        with pytest.raises(StopIteration):
            next(files)
```

Alternative approach. Shouldn’t be too much difference in practice (especially in a test).

Reply:
This is really elegant!